test/chpldoc/globals/variableInitialization.doc.bad

```diff
@@ -0,0 +1,21 @@
+.. default-domain:: chpl
+
+.. module:: variableInitialization.doc
+
+variableInitialization.doc
+====================
```

Comment:
Aww, sorry, looks the new test doesn't run cleanly - this line is generated based on the length of the previous line (so because the name is longer, there need to be more `=`).  Here's an excerpt of the output from start_test:
```
[Executing diff-ignoring-module-line-numbers variableInitialization.doc.bad variableInitialization.doc.comp.out.tmp]
6c6
< ====================
---
> ==========================
[Error matching .bad file for chpldoc/globals/variableInitialization.doc]
```

This should be the last thing you need to do, thanks!

Reply:
Thanks for bearing with my mistakes :D I have pushed the changes as suggested.

Reply:
Oops, the .good file needs to be updated as well!